### PR TITLE
Feature/mww integration

### LINKED ISSUE #1246: Meet with wallet integration
### What would you like to be added?

Easy integration with meetwithwallet.xyz so players can have their calendar on their profiles

### Why is this needed?

Improves coordination by allowing easy scheduling of meetings

### PR DESCRIPTION
**Pull request by githubdoramon**
## Overview

This PR adds the Meet with wallet integration to Meta profiles. It allows players to create and add their public calendar links to their profiles with a simple click

It also adds the Outline variant to Button, as well as correct Blue shadings to theme

Closes #1246

## Implementation

- New API endpoints to do external integration calls without CORS issues
- Extends the External Account types (GITHUB, TWITTER, etc), adding Meet with wallet one

**Side effects**

- Changes schema because of new AccountTpe_Enum item
- Adds some logic to updateSingle backend logic

## Assets

image

image

### CODE REVIEW

**Comment by alalonde** [comment]
When saving my calendar addition, I got some errors from the ceramic instance:
`{"error":"Failed to parse the JSON: SyntaxError: Unexpected token � in JSON at position 0"}`

This was posting to the /streams API

**Comment by githubdoramon** [comment]
> When saving my calendar addition, I got some errors from the ceramic instance: `{"error":"Failed to parse the JSON: SyntaxError: Unexpected token � in JSON at position 0"}`
> 
> This was posting to the /streams API

Hmmm.. .weird, since I don't think the accounts information actually goes to ceramic. @dysbulic tested and didn't have the problem (neither do I). Did you test directly from this branch? Any other hint on something that might be different for you?

**Comment by alalonde** [comment]
> > When saving my calendar addition, I got some errors from the ceramic instance: `{"error":"Failed to parse the JSON: SyntaxError: Unexpected token � in JSON at position 0"}`
> > This was posting to the /streams API
> 
> Hmmm.. .weird, since I don't think the accounts information actually goes to ceramic. @dysbulic tested and didn't have the problem (neither do I). Did you test directly from this branch? Any other hint on something that might be different for you?

Yeah, from here: https://frontend-pr-1250-mjhnbmqqna-uk.a.run.app/players

File: packages/web/components/Player/MeetWithWalletProfileEdition.tsx [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+  // adding a new column to the table just to do the followig, I decided
+  // to this this cast as any to use the information from the form and
+  // don't do bigger changes on the data structure just because of it
+  const profileFields = useProfileField({
+    field: 'meetWithWalletDomain',
+    player,
+    getter: getPlayerMeetwithWalletCalendarUrl,
+  });
+
+  const mwwDomain = (profileFields as any).meetWithWalletDomain || null;
```
[the remark this answers is not in the document]
Comment (githubdoramon) [reply packages/web/components/Player/MeetWithWalletProfileEdition.tsx:50]: From my tests it works normally indeed... And will add the disable next line comment

File: hasura/migrations/1650213538515_adds_mww_account_type/up.sql [hunk 2]
```diff
@@ -0,0 +1 @@
+INSERT INTO "AccountType" (type) VALUES ('MEETWITHWALLET');
```
[the remark this answers is not in the document]
Comment (githubdoramon) [reply hasura/migrations/1650213538515_adds_mww_account_type/up.sql:1]: Added

**Review by alalonde — APPROVED** [review]
Looks good!

**Pull request merged**